ui/shared/components/panel/acmg/AcmgCriteria.jsx

```diff
@@ -0,0 +1,221 @@
+    this.props.setAcmgClassification('Unknown')
+  }
+
+  submitForm = () => {
+    if (this.props.acmgClassification === 'Unknown') {
+      this.setFormWarning('Please select at least one criteria from the table below.')
+    } else if (this.props.acmgClassification === 'Conflicting') {
+      this.setFormWarning('You have conflicting score. Please verify your selections.')
+    } else {
+      this.setFormWarning(false)
+      this.props.setActive(false)
+      this.props.variant.acmgClassification = {
+        score: getNewScoreValue(this.props.criteria),
+        classify: this.props.acmgClassification,
+        criteria: this.props.criteria,
+      }
+      this.props.dispatchUpdateVariantClassification()
+    }
+  }
+
+  getTableRows = () => {
+    return ACMG_DROP_DOWN_OPTIONS.map((dropDownOption) => {
+      let startArray = 0
+      let endArray = TABLE_ROW_TOTAL_COLUMNS
+      const rows = []
+
+      while (startArray !== dropDownOption.options.length) {
+        const row = dropDownOption.options.slice(startArray, endArray)
+        rows.push(row)
+
+        startArray = endArray
+        endArray += TABLE_ROW_TOTAL_COLUMNS
+      }
+
+      const elements = rows.map((row, rowIdx) => {
+        const criteriaUsed = this.getCriteriaUsed()
+        return (
+          <Table.Row>
+            { rowIdx === 0 ? <Table.Cell rowSpan={dropDownOption.optionRowSpan}><span style={FONT_STYLE_WRITING_MODE}>{dropDownOption.optionTitle}</span></Table.Cell> : null }
+            { row.map((cell, cellIdx) => {
+              if (cell.length === 0) {
+                return <Table.Cell />
+              }
+              const color = TABLE_COLUMN_COLOR[cellIdx % TABLE_ROW_TOTAL_COLUMNS]
+              const option = cell[0]
+
+              return (
+                <Table.Cell>
+                  <Table size="small" color={color}>
+                    <Table.Body>
+                      <Table.Row textAlign="center" key={`table-row-${option.description}`}>
+                        <Table.Cell width={1}>{option.key}</Table.Cell>
+                        <Table.Cell width={2}>{option.description}</Table.Cell>
+                        <Table.Cell width={1}>
+                          <Dropdown
+                            value={option.key ? 'Y' : ''}
+                            key={`dropdown-${option.key}`}
+                            options={option.values}
+                            onChange={this.addOrRemoveCriteria}
+                            text={criteriaUsed[option.key] ? 'Y' : 'N'}
+                          />
+                        </Table.Cell>
+                      </Table.Row>
+                    </Table.Body>
+                  </Table>
+                </Table.Cell>
+              )
+          })}
+          </Table.Row>
+        )
+      })
+
+      return elements
+    })
+  }
+
+  getNewScore = (criteria) => {
+    const newScore = getNewScoreValue(criteria)
+    if (newScore >= 10) {
+      return 'Pathogenic'
```

Comment:
It would be more maintainable to define constants rather that hardcoding strings. This would also allow you to use shared constant with, say `getButonBackgroundColor` to keep all the criteria in one place and prevent typos causing bugs in the future as the code evolves. One way you could do it:
```
const CATEGORY_CONFIGS = [
  { category: 'Benign', color: 'blue', threshold: -6 },
  ...
  { category: 'Likely Pathogenic', color: 'orange', threshold: 10 },
  { category: 'Pathogenic', color: 'orange', threshold: null },
]
const CATEGORY_COLORS = CATEGORY_CONFIGS.reduce(
  (acc, { category, color }) => ({ ...acc, [category]: color }), {})

...
getNewScore = (criteria) => {
  const newScore = getNewScoreValue(criteria)
  const { category } = CATEGORY_CONFIGS.find(({ category, threshold }) => 
    threshold === null || newScore < threshold
  )
  return category
```